wp-content/plugins/core/src/Blocks/Patterns/Loop_Pattern.php

```diff
@@ -0,0 +1,46 @@
+<?php declare(strict_types=1);
+
+namespace Tribe\Project\Blocks\Patterns;
+
+/**
+ * Class Block_Pattern_Base
+ *
+ * Edits the block categories.
+ */
+class Loop_Pattern extends Pattern_Base {
+
+	protected string $name = 'tribe/loop_pattern';
+
+	protected function get_args(): array {
+		return [
+			self::TITLE       => esc_html__( 'Loop Pattern', 'tribe' ),
+			self::DESCRIPTION => esc_html__( 'Loop Pattern', 'tribe' ),
+			self::CONTENT     => $this->get_content(),
+			self::CATEGORIES  => [ Pattern_Category::CUSTOM_PATTERN_CATEGORY_SLUG ],
+			self::KEYWORDS    => [ esc_html__( 'loop', 'tribe' ), esc_html__( 'query', 'tribe' ) ],
+		];
+	}
+
+	private function get_content(): string {
+		return '<!-- wp:group {"tagName":"section","className":"c-block b-content-loop l-container"} -->
```

Comment:
@ChrisMKindred would we be able to create a template file for this and return it here using something like `tribe_template_part`? It would be great to make this easier to read and edit.

Reply:
@LayaTaal 
That is an option for sure. I don't know if, in practice, you would ever be interacting with the content directly in your code editor. The way I see it is that Design and Engineering would work together on setting up the blocks on a page. Once everybody is happy with the layout, switch to the Code Editor in the block editor settings and copy the content and paste it into whatever we are using to return the pattern content.